Task_TheFlexGame/index.css

```diff
@@ -1 +1,85 @@
-/* Start coding below for FLEX GAME TASK*/
+.container{
+    width: 2828px;
+    height: 2562px;
+    display: flex;
+    justify-content: space-between;
+}
+.menu{
+    width: 639px;
+    height: 2562px;
+    background: #604AC6;
+    display: flex;
+    flex-direction: column;
+    padding: 84.5px;
+    box-sizing: border-box;
+}
+.content{
+    width: 2145px;
+    height: 2562px;
+    background: #01AB63;
+    padding: 84.5px;
+    display: flex;
+    flex-direction: column;
+    box-sizing: border-box;
+}
+.menu-title{
+    width: 590px;
+    height: 120px;
+    font-family: "Inter", sans-serif;
+    font-size: 112px;
+    font-weight: 600;
+    line-height: 136px;
+    letter-spacing: 0em;
+    text-align: left;
+    color: #FFFFFF;
+
+}
+.menu-boxes{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 470px;
+    height: 254px;
+    background: #D9D9D9;
+    margin-top: 84.5px;
+    font-family: Inter;
+    font-size: 112px;
+    font-weight: 600;
+    line-height: 136px;
+    letter-spacing: 0em;
+    color: #604AC6;
+}
+.content-title{
+    font-family: "Inter", sans-serif;
+    width: 590px;
+    height: 120px;
+    font-size: 112px;
+    font-weight: 600;
+    line-height: 136px;
+    letter-spacing: 0em;
+    text-align: left;
+    color: #FFFFFF;
+}
+.content-boxes{
+    width: 295px;
+    height: 254px;
+    color: white;
+    font-family: "Inter", sans-serif;
+    font-size: 112px;
+    font-weight: 600;
+    line-height: 136px;
+    letter-spacing: 0em;

```

Comment:
Check consistency in text alignment property values between .content-boxes and .content-title classes.